packages/core/src/util/sequelize.ts

```diff
@@ -1,21 +1,37 @@
 import { Sequelize } from "sequelize";
+import { z } from "zod";
+
+export const dbCredsSchema = z.object({
+  dbname: z.string(),
+  username: z.string(),
+  password: z.string(),
+  host: z.string(),
+  port: z.number(),
+  engine: z.literal("postgres"),
+});
 
 /**
  * This function is used to initialize sequelize for lambda functions.
  */
 export function initSequelizeForLambda(dbCreds: string) {
   const sqlDBCreds = JSON.parse(dbCreds);
+  const parsedDbCreds = dbCredsSchema.parse(sqlDBCreds);
 
-  const sequelize = new Sequelize(sqlDBCreds.dbname, sqlDBCreds.username, sqlDBCreds.password, {
-    host: sqlDBCreds.host,
-    port: sqlDBCreds.port,
-    dialect: sqlDBCreds.engine,
-    pool: {
-      max: 5,
-      min: 1,
-      acquire: 30000,
-      idle: 10000,
-    },
-  });
+  const sequelize = new Sequelize(
+    parsedDbCreds.dbname,
+    parsedDbCreds.username,
+    parsedDbCreds.password,
+    {
+      host: parsedDbCreds.host,
+      port: parsedDbCreds.port,
+      dialect: parsedDbCreds.engine,
+      pool: {
+        max: 5,
```

Comment:
Max 5 connections in the pool? Was it always this way?

5 seems a bit low

Reply:
In theory this is just for the lambdas, so not even needed a conn pool

Reply:
https://metriport.slack.com/archives/C065BLRBUDQ/p1709169214821549?thread_ts=1709160304.662419&cid=C065BLRBUDQ